cluster/saltbase/salt/flannel-server/network.json

```diff
@@ -0,0 +1,8 @@
+{
+    "Network": "192.168.0.0/16",
```

Comment:
I propose 10.15.0.0/12 (4K nodes at /24 each or more nodes at smaller per-node scopes) or else 172.16.0.0/12.  I like that 192 or 172 is not 10.*, but I think 192 doesn't leave enough room for high-scale testing, and moving machines to < /26 is bad for the same reason.  I dislike using 172 for this because that is the range docker naturally uses, so this will cause confusion if not actual problems with docker's own bridge.

So I propose 10.15.0.0/12 as the default.